build/crds/sedna.io_federatedlearningjobs.yaml

```diff
@@ -44,6 +44,7 @@ spec:
                   should have
                 properties:
                   model:
+                    description: TrainModel defines model of CR
```

Comment:
Use `federated learning job` instead of `CR`

Reply:
ok